configmaps/appoptics-token.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
```

Comment:
@trevrosen hey, do you think it's a good idea to switch to secret + configmaps here? it might be breaking change but makes this project much more managable

Reply:
I think it's fine to switch, yeah.